### PR TITLE
Statistics for K Points

### LINKED ISSUE #47: Run statistics for the top K points
For the top K points (k to be specified by the user, or set to a default value) with the highest risk, statistics are returned, such as the class distribution, correct or misclassification, distance to class mean (in order to find if they are outlier).

Explanation: Have a function that return a numPy histogram, which for each target label  (10 in CIFER) returns information how many points are in each target label. For example if K=100, we get returned information that 99 points are in one target label and 1 in second, and 0 in other eight target label.

### PR DESCRIPTION
**Pull request by Erdnaf**
@budmil and me added a UserOutput class for the statistics

closes #47

### CODE REVIEW

File: privacy_evaluator/output/user_output.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        self.all_labels = all_labels
+
+    def histogram_top_k(self, k: int = 10, show_diagram: bool = True):
+        """
+        Draws a historgam of the top k points with biggest Privacy score
+        :param k: the number of points to consider, default 10
+        :param show_diagram: determines if the diagram should be shown, default: True
+        :return: All lables of the data with the number of points that are in the top k
+        :rtype: tuple
+        """
```
Comment (marisanest) [inline privacy_evaluator/output/user_output.py:30]: Maybe move this return type to the method declaration (e.g. `histogram_top_k(self, k: int = 10, show_diagram: bool = True) -> Tupel[np.ndarray, np.ndarray] :`)

**Review by marisanest — CHANGES_REQUESTED** [review]
The rest looks very nice :+1: Just one minor change I would do.

File: privacy_evaluator/output/user_output.py (hunk 1 above)
Comment (Erdnaf) [reply privacy_evaluator/output/user_output.py:31]: fixed it

**Review by jtorhoff — APPROVED** [review]

File: privacy_evaluator/output/user_output.py (hunk 1 above)
Comment (marisanest) [reply privacy_evaluator/output/user_output.py:34]: You can then just remove the `:type: ...` line in the docstring.

**Review by marisanest — CHANGES_REQUESTED** [review]

**Review by marisanest — APPROVED** [review]
Well done :smile:

**Review by blauertee — COMMENTED** [review]
Nice! looks good

**Review by blauertee — APPROVED** [review]

**Pull request merged**